pkg/kots/lint.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/replicatedhq/kots-lint/pkg/util"
 	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
 	kotsscheme "github.com/replicatedhq/kots/kotskinds/client/kotsclientset/scheme"
+	"github.com/replicatedhq/kurlkinds/pkg/lint"
```

Comment:
let's call this `kurllint` just to make it clearer
```suggestion
	kurllint "github.com/replicatedhq/kurlkinds/pkg/lint"
```

Reply:
fixed.